### ISSUE TITLE
README: Ingot will not work on Chrome 106 or after

### ISSUE THREAD

**Issue by Nebelung-Dev** [FogNetwork/Ingot#13]
Don't make an issue for this I can not fix it, sorry :(

**Comment by Crushstar** [comment]
What on chrome OS build 106 is stopping ingot. I'm wondering if there is a way around it?

**Comment by RedGuyVR** [comment]
do you know why it stopped working?

**Comment by Gitry305** [comment]
I turned on devmode, and checked the true extension page and it was still off

Also, can you make a policy changer bookmarklet

**Comment by Gitry305** [comment]
as in able to change to policies on chrome://policy page

**Comment by Craexz** [comment]
I used this for a couple days then it got patched with the new chrome version, is there any alternative to this? anything that has the same effect, just disabling extensions.

**Comment by bohrium2b** [comment]
It seems that a Content Security Policy is blocking this from working. 
![image](https://user-images.githubusercontent.com/85321730/198395143-4bea34b3-71c4-40f2-89eb-cbd6212e723d.png)

**Comment by Craexz** [comment]
> It seems that a Content Security Policy is blocking this from working. ![image](https://user-images.githubusercontent.com/85321730/198395143-4bea34b3-71c4-40f2-89eb-cbd6212e723d.png)

Maybe if the script was actually within the bookmarklet instead of fetching it from that site it would work?

**Comment by sylvxa** [comment]
The loophole LTBEEF and Ingot used was patched in Chrome 106, it is not a CSP issue or anything and unfortunately, unless there is another "exploit" found they won't work for a while.

**Comment by Gitry305** [comment]
whelp, i have a recommendation. create something of a policy destroyer

**Comment by DodoSeal** [comment]
I'm on version 102. My friend is also on version 102. Our chromebooks have no differences in policies, versions, model. Somehow this works on his device and nobody else's. If I log into my account on his it still works.

**Comment by DodoSeal** [comment]
> whelp, i have a recommendation. create something of a policy destroyer
> […](#)

That's not possible...

**Comment by raventhecrabraver** [comment]
> > whelp, i have a recommendation. create something of a policy destroyer
> > […](#)
> 
> That's not possible...

dude dont say its not possible you dont know that for sure.

**Comment by DodoSeal** [comment]
> > > whelp, i have a recommendation. create something of a policy destroyer
> 
> > > […](#)
> 
> > 
> 
> > That's not possible...
> 
> 
> 
> dude dont say its not possible you dont know that for sure.

Are you the one with the experience here. Just think, Google is a billion dollar company. They have extremely smart engineers.

**Comment by Aka-unblock** [comment]
Here's what you have to do. Make a bookmark with chrome://restart. Next, click on it and your Chromebook should restart, AS SOON AS IT COMES BACK UP CLICK THE CALENDER AT THE BOTTOM OF THE SCREEN and it will restart again then everything will be unblocked if you don't have a calendar at the bottom of the screen, go to chrome://flags and look up a calendar and enable anything that's there. It works for me if it doesn't work for you, I'm sorry but this works!

**Comment by miiuzu** [comment]
> Here's what you do, make a bookmark with chrome://restart in it and name it something like Restart then make another bookmark with chrome://signOut the o has to be capitalized then click the signOut button first and press the blue button then press the restart and wait for it to load up, and your good!

Hey! so I've tried that it apparently doesn't _work at all._  what is the blue button?

**Comment by Antgod4** [comment]
There is a bypass for iboss https://spacesaver.github.io/Ingot-for-iBoss maybe you could add that to your bypassing list the repository is https://github.com/SpaceSaver/Ingot-for-iBoss.

**Comment by miiuzu** [comment]
thanks!!

**Comment by Aka-unblock** [comment]
> It seems that a Content Security Policy is blocking this from working. ![image](https://user-images.githubusercontent.com/85321730/198395143-4bea34b3-71c4-40f2-89eb-cbd6212e723d.png)

Literally csp shit, you can't just load scripts, fetch doesn't work either.

**Comment by Aka-unblock** [comment]
> > > > whelp, i have a recommendation. create something of a policy destroyer
> > 
> > 
> > > > […](#)
> > 
> > 
> > > 
> > 
> > 
> > > That's not possible...
> > 
> > 
> > dude dont say its not possible you dont know that for sure.
> 
> Are you the one with the experience here. Just think, Google is a billion dollar company. They have extremely smart engineers.

What I'm saying, no IQ, you can't just edit policies, bro is straight up stupid.

**Comment by Aka-unblock** [comment]
> The loophole LTBEEF and Ingot used was patched in Chrome 106, it is not a CSP issue or anything and unfortunately, unless there is another "exploit" found they won't work for a while.

Nobody said it was about CSPs, it was a Chrome mistake involving the API management, this one was the only API, that can fuck up extensions how it did with ltbeef, there are many more exploits, just you can't delete
extensions without management, it is intended behavior, without management you wouldn't be able to delete extensions and more, Chrome needed the management API, Chrome didn't expect people would use that API, for bypassing.

**Comment by Aka-unblock** [comment]
> do you know why it stopped working?

Yes, when Chrome added the management API, they didn't specify which extensions the webstore could disable, this also included *admin ones*. When Chrome realized their error, they made it so the webstore couldn't just disable an admin extensions, and this will most likely stay as there is no problems with this, you're not supposed to be able to delete admin extensions in the first place.
